sdk/maps/maps-search/samples-dev/search.ts

```diff
@@ -0,0 +1,349 @@
+      maxDetourTime,
+      searchAlongRoute,
+      searchAlongRouteOptions
+    )
+  );
+
+  console.log(" --- Search inside geometry: single polygon");
+  const searchInsideGeometryQuery = "burger";
+  const searchGeometryPolygon: GeoJsonPolygon = {
+    type: "Polygon",
+    coordinates: [
+      [
+        [-122.43576049804686, 37.7524152343544],
+        [-122.43301391601562, 37.70660472542312],
+        [-122.36434936523438, 37.712059855877314],
+        [-122.43576049804686, 37.7524152343544],
+      ],
+    ],
+  };
+  const searchInsideGeometryOptions = { top: 2 };
+  console.log(
+    await client.searchInsideGeometry(
+      searchInsideGeometryQuery,
+      searchGeometryPolygon,
+      searchInsideGeometryOptions
+    )
+  );
+
+  console.log(" --- Search inside geometry: multiple polygons");
+  const searchGeometryPolygons: GeoJsonPolygonCollection = {
+    type: "GeometryCollection",
+    geometries: [
+      {
+        type: "Polygon",
+        coordinates: [
+          [
+            [-122.43576049804686, 37.7524152343544],
+            [-122.43301391601562, 37.70660472542312],
+            [-122.36434936523438, 37.712059855877314],
+            [-122.43576049804686, 37.7524152343544],
+          ],
+        ],
+      },
+      {
+        type: "Polygon",
+        coordinates: [
+          [
+            [-121.43576049804686, 38.7524152343544],
+            [-121.43301391601562, 38.70660472542312],
+            [-121.36434936523438, 38.712059855877314],
+            [-121.43576049804686, 38.7524152343544],
+          ],
+        ],
+      },
+    ],
+  };
+  console.log(
+    await client.searchInsideGeometry(
+      searchInsideGeometryQuery,
+      searchGeometryPolygons,
+      searchInsideGeometryOptions
+    )
+  );
+
+  console.log(" --- Search inside geometry: circles and polygons");
+  const searchGeometryPolygonsOrCircles: GeoJsonCircleOrPolygonFeatureCollection = {
+    type: "FeatureCollection",
+    features: [
+      {
+        type: "Feature",
+        geometry: {
+          type: "Polygon",
+          coordinates: [
+            [
+              [-121.43576049804686, 38.7524152343544],
+              [-121.43301391601562, 38.70660472542312],
+              [-121.36434936523438, 38.712059855877314],
+              [-121.43576049804686, 38.7524152343544],
+            ],
+          ],
+        },
+        properties: {},
+      },
+      {
+        type: "Feature",
+        geometry: {
+          type: "Point",
+          coordinates: [-121.43576049804686, 38.7524152343544],
+        },
+        properties: {
+          subType: "Circle",
+          radius: 5000,
+        },
+      },
+    ],
+  };
+  console.log(
+    await client.searchInsideGeometry(
+      searchInsideGeometryQuery,
+      searchGeometryPolygonsOrCircles,
+      searchInsideGeometryOptions
+    )
+  );
+
+  console.log(" --- Search address batch (long-running):");
+  const searchAddressRequests: SearchAddressRequest[] = [
```

Comment:
Looks that these batch examples overlap with `batchRequest.ts`?

Reply:
Yes. But I would like to leave it here to give an overview to the users of the API in `search.ts`. The main purpose for `batchRequest.ts` is to demonstrate LRO API like `resume*`, `getResults()`, `toString()`...